ranger/blackhole_vm_metadata_server.sh

```diff
@@ -0,0 +1,17 @@
+set -xe
+
+# Blackhole the VM metadata server except for root.
```

Comment:
May we use something more obvious then "blackhole"?

Reply:
`Block` maybe?

Reply:
done